apps/verticalhome/js/app.js

```diff
@@ -219,6 +221,12 @@
           window.removeEventListener('gaiagrid-layout-ready', this);
           break;
 
+        case 'visibilitychange':
+          setTimeout(function() {
+            document.body.style.overflow = document.hidden ? 'hidden' : '';
```

Comment:
One last comment about this - we should add a comment as to what this is doing (hiding overflow to stop displayport rendering). Also, is it necessary for it to be in a timeout?